README.md

```diff
@@ -1,7 +1,9 @@
+
 # AssignmentDesignPatterns
 
 Design pattern identified in the CellPhone Program
 
+
 1) Factory Pattern
```

Comment:
Wouldn't you need a factory subclass for each individual mobile?

Reply:
CellPhoneFactory is actually providing both the creation of Android and Ios objects and returning the concrete class objects as per the users choice.